### PR TITLE
Add more translation keys

### PR DESCRIPTION
**Pull request by margarizaldi**

### CODE REVIEW

File: resources/views/components/modals/curator-curation.blade.php [hunk 1]
```diff
@@ -46,19 +46,19 @@ class="h-full"
...
-                            <x-curator::curation-input type="text" prefix="Height" suffix="px" name="height" x-on:input="setCropBoxHeight($event)" x-bind:value="Math.round(cropBoxData.height)" />
-                            <x-curator::curation-input type="text" prefix="Rotate" suffix="deg" name="rotate" x-on:input="cropper.rotateTo($event.target.value)" x-bind:value="data.rotate" />
+                            <x-curator::curation-input type="text" prefix="{{ __('curator::views.curation.width') }}" suffix="px" name="width" x-on:input="setCropBoxWidth($event)" x-bind:value="Math.round(cropBoxData.width)" />
+                            <x-curator::curation-input type="text" prefix="{{ __('curator::views.curation.height') }}" suffix="px" name="height" x-on:input="setCropBoxHeight($event)" x-bind:value="Math.round(cropBoxData.height)" />
+                            <x-curator::curation-input type="text" prefix="{{ __('curator::views.curation.rotate') }}" suffix="{{ __('curator::views.curation.rotate_deg') }}" name="rotate" x-on:input="cropper.rotateTo($event.target.value)" x-bind:value="data.rotate" />
```
Comment (margarizaldi) [inline resources/views/components/modals/curator-curation.blade.php:61]: As only "deg" word isn't used in my language, while "px" should be (I guess) used in every language.

Comment (awcodes) [reply resources/views/components/modals/curator-curation.blade.php:61]: 'deg' is just an abbreviation for 'degree'. As in 90°

Comment (margarizaldi) [reply resources/views/components/modals/curator-curation.blade.php:61]: Yes sure, I mean the word 'deg' must be translatable, so we can use local abbr or ° symbol... While pixel (px) is not necessary to be translatable I think...

Comment (awcodes) [reply resources/views/components/modals/curator-curation.blade.php:61]: I agree, no need to translate 'px'. I think that's pretty universal.

**Pull request merged**